src/arviz_plots/plot_collection.py

```diff
@@ -337,6 +337,13 @@ def show(self):
         else:
             self.viz["plot"].item()
 
+    def savefig(self, *args, **kwargs):
```

Comment:
given the backend versions have signature `savefig(chart, filename, **kwargs)` the *args part can only ever be `filename` so I would match that here.

Also, seeing the tests with a `str(path)`, would it be possible to support both string and path objects as `filename`? I think if we convert the input to `pathlib.Path` we can use `filename.stem == ".html/png"` instead of `.endswith` for checking the extension